README.md

```diff
@@ -1 +1,10 @@
-# hw01
\ No newline at end of file
+### Self-Introduction  
+
+![](images/profile_pic.png)  
+
+Hi! My name is **Izzy Allum**, and I am a first year at the University of Chicago. I'm from Washington, DC, so that's where I've been spending quaratine. My hobbies include:  
```

Comment:
Are you actually DC, or just the DMV? I'm originally from Loudoun County.

Also, I have no idea when the "DMV" became a thing.

Reply:
actually DC!